src/hooks/actions.tsx

```diff
@@ -281,12 +284,13 @@ export function useProposeAction({
       success: <>Proposed price</>,
       error: <>Failed to propose price</>,
     })
-      .then((receipt) => {
-        if (receipt && query)
-          oracleEthersApis?.[query.oracleType]?.[
-            chainId
-          ]?.updateFromTransactionReceipt(receipt);
-      })
+      // TODO: this is disabled because wagmi move from ethers, should look into updating this eventually
```

Comment:
wagmi switch from ethers broke interop between the fe and fetching services in libs which still use ethers